### ISSUE TITLE
Task: Write policy/process for "unofficial" PyLadies and make them official

### ISSUE THREAD

**Issue by Mariatta** [pyladies/project-admin-logisitics#11]
# TL;DR

The policy should answer the following questions:
1. Why is this policy necessary?
2. What are "unofficial" PyLadies
3. How do we make them official?
4. For accountability, how do we (administrators) document this process?

# Details

## 1. Why is this policy necessary?

We noticed there are a number of PyLadies communities without their own pyladies.com email address, or not listed in "Initial PyLadies interest form"

PyLadies is trademarked and we need to protect our brand.

A PyLadies chapter uses some kind of resource in various forms: for example subscription to Meetup Pro account, they may qualify for grants from The PSF, etc. We may not be able to provide support and resource if a chapter is not official PyLadies chapter.

If they do not have @pyladies.com email address, and we don't have the organizers contact info in "initial PyLadies interest form", we have no way to get in touch with them.

## 2. What are "unofficial" PyLadies?

If they call themselves PyLadies, but they don't have a @pyladies.com email address.
Our expectation is all social media and meetup accounts are associated with @pyladies.com email address instead of a personal email address.

## 3. How do we make them official?

We'll happily create their own @pyladies.com email address, however we would ask for the following:
- their current organizers name and email addresses
- after they received their own @pyladies.com email address, they need to start using it, (e.g. change their meetup/twitter/etc from personal email address to @pyladies.com email address)

They should also adhere to other general guidelines for a PyLadies community (e.g. agree to our code of conduct, etc)

We may want to set a deadline, like 3 months for them to complete the transition. (to change email address etc). 

Question: what if they refuse/non-responsive? Do we take actions?

## 4. For accountability, how do we (administrators) document this process?

Tbd

**Comment by Mariatta** [comment]
All of this is related to "New chapter policy" (https://github.com/pyladies/project-admin-logisitics/issues/9). They should have an understanding of what it means to be an official PyLadies, what kind of resource and support they can get as a PyLadies chapter, and what are their expected role and responsibilities.

**Comment by terezaif** [comment]
for the new chapter form should we add the option of "this chapter has actually been running for a while but unregistered" ? I think labelling it in public with unofficial would make people feel they did something very wrong.. and we want to get them to submit the info..



maybe even in the name New Chapter (or unregistered) Form

then we could close this issue

**Comment by Mariatta** [comment]
I was thinking , if it is a pre-existing unregistered chapter that's actually active, perhaps we don't need to ask as many questions. Probably we just need the contact info of all organizers, links to their meetup/social media/website, and their agreement to changing those to their pyladies one.
So I think it should be a new separate form.
